src/structure/data.ts

```diff
@@ -5,8 +5,8 @@ import { ValenceModel } from '../chemistry/valence-model'
 
 export interface Data {
   structure: Structure
-  '@spatialLookup': SpatialHash | undefined
-  '@valenceModel': ValenceModel | undefined
+  '@spatialLookup': SpatialHash | undefined | any
```

Comment:
Why the added `any`?

Reply:
Without those `| any`, I get this error:
```
> tsc && cpx "src/**/*.{vert,frag,glsl}" build/js/src/ && cpx "lib/*.js" build/js/lib/ && cpx "package.json" build/js/ && cpx "src/polyfills.js" build/js/src/

src/structure/data.ts(23,3): error TS2322: Type 'SpatialHash' is not assignable to type 'undefined'.
src/structure/data.ts(30,3): error TS2322: Type '{ charge: Int8Array; implicitH: Int8Array; totalH: Int8Array; idealGeometry: Int8Array; }' is not assignable to type 'undefined'.
npm ERR! code ELIFECYCLE
```

I doubt the `any` is the right solution, but it did work (build & run OK).